pulp_ansible/tests/functional/api/collection/v3/test_serializers.py

```diff
@@ -32,61 +30,62 @@ def setUpClass(cls):
-        monitor_task(sync_response.task)
-        repo = self.repo_api.read(repo.pulp_href)
-
-        # Create a distribution.
-        body = gen_distribution()
-        body["repository"] = repo.pulp_href
-        distribution_create = self.distributions_api.create(body)
-        created_resources = monitor_task(distribution_create.task).created_resources
-        distribution = self.distributions_api.read(created_resources[0])
-        self.base_path = distribution.base_path
-        self.addCleanup(self.distributions_api.delete, distribution.pulp_href)
-
     @unittest.skip("FIXME: Re-enable later")
     def test_v3_updated_at(self):
         """Test Collections V3 endpoint field: ``updated_at``."""
-        repo = self.repo_api.read(self.repo_href)
+        body = gen_ansible_remote(
+            url="https://galaxy.ansible.com",
+            requirements_file="collections:\n  - testing.k8s_demo_collection",
+        )
+        remote = self.remote_collection_api.create(body)
+        self.addCleanup(self.remote_collection_api.delete, remote.pulp_href)
 
-        collections = self.collections_api.list(self.base_path)
+        repo = self._create_repo_and_sync_with_remote(remote)
+        distribution = self._create_distribution_from_repo(repo)
+
+        collections = self.collections_api.list(distribution.base_path)
 
         original_highest_version = collections.data[0].highest_version["version"]
-        original_updated_at = datetime.strptime(
-            collections.data[0].updated_at, "%Y-%m-%dT%H:%M:%S.%fZ"
-        )
+        original_updated_at = collections.data[0].updated_at
 
         versions = self.collections_versions_api.list(version="3.6.4")
         original_total_versions = self.collections_versions_v3api.list(
-            "pulp_installer", "pulp", self.base_path
-        ).meta["count"]
+            "pulp_installer", "pulp", distribution.base_path
+        ).meta.count
 
         data = {"remove_content_units": [versions.results[0].pulp_href]}
         response = self.repo_api.modify(repo.pulp_href, data)
         monitor_task(response.task)
 
-        collections = self.collections_api.list(self.base_path)
+        collections = self.collections_api.list(distribution.base_path)
         highest_version = collections.data[0].highest_version["version"]
-        updated_at = datetime.strptime(collections.data[0].updated_at, "%Y-%m-%dT%H:%M:%S.%fZ")
+        updated_at = collections.data[0].updated_at
 
         total_versions = self.collections_versions_v3api.list(
-            "pulp_installer", "pulp", self.base_path
-        ).meta["count"]
+            "pulp_installer", "pulp", distribution.base_path
+        ).meta.count
 
         self.assertEqual(highest_version, original_highest_version)
         self.assertEqual(original_total_versions, total_versions + 1)
         self.assertGreater(updated_at, original_updated_at)
+
+    def test_v3_collection_version(self):
```

Comment:
This test is great. It's really only specific to data populated by sync so maybe:

```suggestion
    def test_v3_collection_version_from_synced_data(self):
```

Reply:
What would be really great is if we could also use the CLI to publish a collection and then make similar assertions. That codepath flows through the importer so only a test like that can assert on our integration code with the importer. Wdyt?

Reply:
Actually maybe the existing CLI upload test could just use a collection that has all this data and then make assertions in that existing test.